reference/events/storage_events.rst

```diff
@@ -0,0 +1,91 @@
+Storage events
+==============
+
+Storage events are dispatched in the PIM when you manipulate data using the savers and removers services.
+They are basicly usefull if you want to add your business logic before or after an object is saved or removed.
+
+Here is a list of all the events:
+
+``akeneo.storage.pre_remove``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Event Class: `RemoveEvent <https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Component/StorageUtils/Event/RemoveEvent.php>`_
+
+This event is dispatched before we remove an object using a remover.
+
+These are the built-in PIM subscribers registered to this event:
+
+================================================================  ===============
+Listener Class Name                                               Priority
+================================================================  ===============
+`CheckChannelsOnDeletionSubscriber`_                              0
+`ProductTemplateAttributeSubscriber`_                             0
+`RuleRelationSubscriber (Enterprise edition)`_                    0
+`RemoveChannelSubscriber (Enterprise edition)`_                   0
+`CheckPublishedProductOnRemovalSubscriber (Enterprise edition)`_  0
+================================================================  ===============
+
+.. _CheckChannelsOnDeletionSubscriber: https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/CatalogBundle/EventSubscriber/Category/CheckChannelsOnDeletionSubscriber.php
+.. _ProductTemplateAttributeSubscriber: https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/CatalogBundle/EventSubscriber/ProductTemplateAttributeSubscriber.php
+.. _RuleRelationSubscriber (Enterprise edition): https://github.com/akeneo/pim-enterprise-dev/blob/master/src/PimEnterprise/Bundle/CatalogRuleBundle/EventSubscriber/RuleRelationSubscriber.php
+.. _RemoveChannelSubscriber (Enterprise edition): https://github.com/akeneo/pim-enterprise-dev/blob/master/src/PimEnterprise/Bundle/ProductAssetBundle/EventSubscriber/ORM/RemoveChannelSubscriber.php
+.. _CheckPublishedProductOnRemovalSubscriber (Enterprise edition): https://github.com/akeneo/pim-enterprise-dev/blob/master/src/PimEnterprise/Bundle/WorkflowBundle/EventSubscriber/PublishedProduct/CheckPublishedProductOnRemovalSubscriber.php
```

Comment:
What should I do then ? I did try to remove the link and display the full file path before and it adds an horizontal scroll that kills the user experience.